core/templates/components/state-editor/state-skill-editor/state-skill-editor.directive.html

```diff
@@ -0,0 +1,107 @@
+<md-card class="oppia-editor-card-with-avatar" id="tutorialStateSkill">
+  <div>
+    <div class="oppia-exp-skill-card-header-container oppia-mobile-collapsible-card-header"
+         ng-click="toggleSkillEditor()">
+      <h3 class="oppia-exp-skill-card-header">
+        Skill
+      </h3>
+      <i class="fa fa-caret-down" ng-if="!skillCardIsShown" aria-hidden="true">
+      </i>
+      <i class="fa fa-caret-up" ng-if="skillCardIsShown" aria-hidden="true">
+      </i>
+    </div>
+    <div class="oppia-add-skill-button-container" ng-if="skillCardIsShown">
+      <div ng-if="!StateSkillService.displayed">
+        <button type="button"
+                class="btn btn-secondary oppia-add-skill-button protractor-test-open-add-skill-modal"
+                ng-click="addSkillModalAsync()">
```

Comment:
I don't understand this method name. What does it mean to "add" a skill modal?

Reply:
Modified.